opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -370,12 +370,14 @@ class ConsoleSpanExporter(SpanExporter):
 
     def __init__(
         self,
+        service_name=None,
         out: typing.IO = sys.stdout,
         formatter: typing.Callable[[Span], str] = lambda span: span.to_json()
         + os.linesep,
     ):
         self.out = out
         self.formatter = formatter
+        self.service_name = service_name
 
     def export(self, spans: typing.Sequence[Span]) -> SpanExportResult:
```

Comment:
Should we include the `service_name` in the exported span?

Reply:
The tox-e lint is failing with an error "unused argument service_name". Hence I had to include it. Not sure whether there is any other way to bypass the error.

Reply:
That's fine, I'm suggesting we SHOULD use it in the exported span, since we are providing the option to add it in the constructor.

Reply:
@lzchen , looks like I misunderstood

![opentelemetry_auto_instru_example](https://user-images.githubusercontent.com/55284676/100989511-f3f0f500-3576-11eb-9842-84be9b574a93.PNG)

I have attached a snapshot of the code change  and output at the client and server side.
Please let me know if I understood your suggestion.

Adding the following line of code in export function of ConsoleSpanExporter prints the service_name at the server output.
`
           span_json = json.loads(self.formatter(span))
            span_json.update({'service_name': self.service_name})
`

Reply:
Yes that was what I wanted.

Reply:
@lzchen , because of the above change , the tests related to ConsoleSpanExporter are failing.
Can i modify and include it in the same PR?

Reply:
Yes! If you could modify the tests that would be great!